### ISSUE TITLE
License for PepperArms Asset

### ISSUE THREAD

**Issue by pauloabelha** [leapmotion/LeapUnreal#10]
Dear LeapMotion team,

I want to use the PepperArms asset in an Unreal project for academic research. What is the license on this asset? 
I am planning on creating a new left version asset from the right PeperArms and use it in my project.

Thank you,

**Comment by testmasterbright** [comment]
What version of the Unreal Plugin are you working with?

**Comment by pauloabelha** [comment]
Hi @testmasterbright,

I am currently using version 2.17.1 from https://github.com/leapmotion/LeapUnreal/releases.
However, I found the PepperArms asset in an old release for which I cannot find the link now.
I have the arms working with version 2.17.1. I was wondering what sort of license is there on this asset if I want to publish academic papers and datasets that use this asset.

Thank you and kind regards,
Paulo

**Comment by testmasterbright** [comment]
Hey @pauloabelha, 

Unfortunately we are no longer licensing that asset, which is why you can no longer find it in the older releases. Hopefully this does not hinder you that much and we sincerely apologize if it does. 

Thanks, 
Matt Bright
Leap Motion QA/Release Manager

**Comment by pauloabelha** [comment]
Hi @testmasterbright ,

Does that mean this asset cannot be used even for research purposes?

**Comment by testmasterbright** [comment]
Hi @pauloabelha, 

You can use it in your research project but it must follow the SDK agreement, specifically clause 1.1 and section 2.2.3.

SDK Agreement: https://central.leapmotion.com/agreements/SdkAgreement

"1.1. Development License. Conditioned upon compliance with the terms and conditions of this Agreement, Leap Motion hereby grants you a limited, non-exclusive, personal, revocable, non-sublicensable and non-transferable license to: (a) install and use a reasonable number of copies of the SDK on computers owned or controlled by you for the purpose of developing and testing applications that are **intended for use solely in connection with a Leap Motion Device**, and Leap Motion Software (“Application”); and (b) modify and incorporate into your Application any sample code provided in the SDK."

"2.2.3. You may not, and may not enable others to, distribute the Non-Redistributable Materials."

Let us know if you have any questions.

**Comment by pauloabelha** [comment]
Hi @testmasterbright ,

Thank you for the response.
I want to use this asset to capture a dataset of RGB-D frames of hand-object interaction.
My dataset would only contain the projected 2D images of this asset and it would not in anyway distribute the asset itself in its skeleton, mesh, texture form. The dataset would be made public (I haven't decided on the license of the dataset itself). In this case, am I allowed to proceed and distribute the dataset?

tl; dr I will not distribute the asset, but only 2D images of the asset as it interacts with objects in a virtual world. Is that ok?

**Comment by testmasterbright** [comment]
Hi @pauloabelha, 

Sorry but yes this is a breach of clause 1.1. Let us know if you have any questions.

Thanks, 
Matt Bright
Leap Motion QA Manager

**Comment by pauloabelha** [comment]
Hi @testmasterbright ,

Please tell me if it would be better for me to contact LeapMotion through other media. I jsut opened an issue here out of convenience.
Is a model trained using data collected through LeapMotion considered an application? In other words, can I collect data with LeapMotion, use it to train a model, and then share the trained model?

Thank you again,
Paulo

**Comment by testmasterbright** [comment]
Hey @pauloabelha, 

Unfortunately that does not count. The Leap Motion Controller must be featured as input in the application being shared. 

Thanks, 
Matt
